congame-doc/introduction.scrbl

```diff
@@ -1,37 +1,530 @@
+
+@margin-note{For fun, try entering a non-numeric value in the “age” text box. What happens when you
+click @onscreen{Submit}?}
+
+This will bring you to the final step in the study:
+
+@browser{
+  @bold{@larger{Good Job, Joel}}
+ 
+  Thank you for participating in our survey despite being 11 years old.
+}
+
+That’s it! You can see that the values you entered in the previous step have carried over and are
+being used in the text displayed in this step.
+
+@margin-note{When you upload and run the study on the Congame server, values collected and stored in
+variables created with @racket[defvar] and similar forms get recorded in the server’s database and
+comprise the results of the study.}
+
+You’ve now observed a few more basic concepts first-hand:
+
+@itemlist[
+
+@item{You use @racket[defvar] to define the discrete pieces of information you want to collect from
+study participants.}
+
+@item{You can create multiple @tech{steps} for your study through repeated use of @racket[defstep].}
+
+@item{Within steps, you can insert input controls like @racket[input-text] inside a @racket[form] to
+collect info from participants, and save that info into your @racket[defvar] variables.}
+
+@item{You use the @racket[defstudy] form to define your study as a whole. Within that form, you use
+@racketid[-->] to tell Congame how the steps connect to each other in sequences.}
+
+]
+
+It’s possible to create much more advanced studies than those we have built so far: studies with
+conditional branching, a variety of form controls, timers, randomizing assignments, etc. But what
+you have seen so far gives you the basic framework.
+
+@;===============================================
+
+@section{Uploading and running your study}
+
+When you have finished designing your study, it's time to go live and start collecting responses
+from participants. We’re going to do that now with the @filepath{age-survey.rkt} study we just
+created.
+
+In order to do this, you need access to a Congame server. For the steps that follow, we'll assume:
+
+@itemlist[#:style 'ordered
+
+@item{You have a link/address to a running Congame server.}
+
+@item{You already have a researcher or an administrator account on that server.}
+
+]
+
+@margin-note{See @seclink["congame-setup"] if you need to know how to prepare a server for use with
+studies.} 
+
+
+@; ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+@subsection{Preparing the study: from @racketmodname[conscript/local] to @racketmodname[conscript]}
+
+@bold{Important:} before uploading the study to a Congame server, we need to change it to use 
+@code{#lang conscript} instead of @code{#lang conscript/local}.
+
+The @racketmodname[conscript/local] environment is only useful for local testing; it’s quick to use,
+but it doesn’t permanently record any of the data collected, and cannot differentiate between
+participants. Using @racketmodname[conscript] @mark{unlocks access to the full server’s participant
```

Comment:
What #lang conscript does is that it requires a congame server with DB to be run as a study - locally or online - which stores participant responses. Not sure what "unlocks access" would mean, you need to have an account on a server to do it, but then you can run such a study.

Reply:
I've commented out the whole section about changing `conscript/local` to `conscript`, which includes this section. Once I test that removing `/local` isn't needed on HEAD then I’ll delete it entirely.